addons/web/controllers/main.py

```diff
@@ -505,11 +506,25 @@ def web_login(self, redirect=None, **kw):
         if not odoo.tools.config['list_db']:
             values['disable_database_manager'] = True
 
+        # otherwise no real way to test debug mode in template as ?debug =>
+        # values['debug'] = '' but that's also the fallback value when of
+        # missing variables in qweb
+        if 'debug' in values:
+            values['debug'] = True
+
         response = request.render('web.login', values)
         response.headers['X-Frame-Options'] = 'DENY'
         return response
 
+    @http.route('/web/become', type='http', auth='user', sitemap=False)
```

Comment:
`/web/become-superuser` would be more self-explaining :)

Reply:
also, I'd expect this to work w/ POST request

Reply:
It might allow impersonating other users in the future, I see no reason to be more opinionated about the name choice.

Reply:
aaaand this makes also sense but then I could argue that it could be "/web/become-user" :stuck_out_tongue_winking_eye: 

sounds good then :+1: